### PR TITLE
We hide the __consumer_offsets topic; omit it in topic count

### LINKED ISSUE #108: kafka:info topic count includes __consumer_offsets
There's a disparity between the `kafka:info` topic count and `kafka:topics`. I assume this is because we're hiding `__consumer_offsets` in `kafka:topics`, but including it in the `kafka:info` topic count. We should exclude it from both for consistency.

image

### PR DESCRIPTION
**Pull request by uhoh-itsmaciek**
Fixes #108

### CODE REVIEW

File: commands/info.js [hunk 1]
```diff
@@ -45,9 +45,11 @@ function formatInfo (cluster) {
     })
   }
 
+  // we hide __consumer_offsets; don't count it
+  const topicCount = Math.max(cluster.cluster.topics.length - 1, 0)
```
Comment (uhoh-itsmaciek) [inline commands/info.js]: `Math.max` just in case the topic list ends up empty...

**Comment by tcrayford** [comment]
:+1:

File: commands/info.js (hunk 1 above)
Comment (thody) [inline commands/info.js]: Sorry, late to the game here. We may want to be more explicit about not counting the `__consumer_offsets` topic to future proof for a world where the topic may not be exposed via this API at all.

Comment (uhoh-itsmaciek) [inline commands/info.js]: Good call.

**Pull request merged**